tests/TestResponse.php

```diff
@@ -94,6 +95,17 @@ public function assertFlashContains($flash)
         return $this;
     }
 
+    public function assertNoFlashSet()
+    {
+        Assert::assertNull($this->app['session']->get('flash'));
+    }
+
+    public function assertTargetURLContains($targetUrl)
```

Comment:
What do you think about adding a type declaration here?

Reply:
Good catch, didn't think about that, i'll update it in a second